src/openrct2/ride/Track.h

```diff
@@ -22,6 +22,8 @@ constexpr uint8_t kRCT2DefaultBlockBrakeSpeed = 2;
 constexpr int32_t kBlockBrakeBaseSpeed = 0x20364;
 constexpr int32_t kBlockBrakeSpeedOffset = kBlockBrakeBaseSpeed - (kRCT2DefaultBlockBrakeSpeed << 16);
 
+constexpr uint8_t kMaximumBrakeSpeed = 30;
```

Comment:
Where does this value come from?

Reply:
This was formerly declared on line 1338 of RideConstruction.cpp, as a local variable, and not checked in the Game Action.